code/.env.sample

```diff
@@ -1,5 +1,21 @@
 ETH_GANACHE_MNEMONIC='filter group there hunt fitness junior ghost park route jar entire clown allow rifle meadow'
-
+ETH_GANACHE_MNEMONIC='filter group there hunt fitness junior ghost park route jar entire clown allow rifle meadow'
+ETH_GANACHE_KEY=0x42957a7502370821e1a1f0f59dcb484b9f5cd34bfd79a7c047f0fd7920c8a621
+ETH_GANACHE_KEY_ALICE=0xed640862873aafea56b1cf7c02785f7f4690bc35efbcbde1d93d4e1b2c810501
+ETH_GANACHE_KEY_BOB=0xa000555b277a3104b9c6eb291b95ffd23c4feee4e718045590cef9d7dae004eb
+ETH_GANACHE_KEY_CAROL=0xbe5f2d9baa257fb61a04d3e055ab371b60bdbe61ac438dbc8ace37c491158397
+ETH_GANACHE_KEY_DORA=0xcd93c08b1a23589ad9ecba93183506c38d1d6408691eee3cb719789b3f9d9b92
+ETH_GANACHE_KEY_ERNIE=0x7a1107cef16e5885adfb807a251d03be96673e7f6c642fd17bfe31e01c709cb1
+ETH_GANACHE_KEY_FRANK=0x6a0dac98d0225bf0fce1ad03ced49ae3f2148c866517306429fb9b5b0867784e
+ETH_GANACHE_KEY_GAYLE=0x8f7fe3f3dfce5f135042782aba16cbc8154e74b0e50452ac9cfee0684ed2bf06
+ETH_GANACHE_KEY_HUBERT=0x4eee9772d4ce0ee499454be093519b8cbd72b2847370a0297296ee270130b355
+ETH_GANACHE_KEY_IVAN=0xc80cd2317e7da687e36262b4d0136aea96a50aa37b9418ba4c5cf85fbd548752
+RELAYER_ADDRESSES='0xC8cD0C9Ca68b853F73917c36E9276770A8D8E4E0,0xfBe4dAdBbDB04Bf3CAbA788D9a7eb23e55313b1F,0xE0b00c60B17cB7987aD79F04Efd7e2572C4a6100,0xe76433Fe7261bDe3e0bbb18F12fE01d0D424E9cD,0xbbf2950F6Bf37928b83AB6d0D183fE6302238AC5,0xc292a5cf79b4942E489f74Ce9997A726789Cd5ED,0xe4f9c8318E6861e38bF65Ff494BABD5D3a4573a8,0xe60Ba21dbee59E4BCA4489E049313CC4603aeA17,0xE55201A6c90c75AC3d6aB8855E749D00807E26a4,0x1b494Fd8Fec1094f4515d5793DbCD70a457e88Ed,0xf8440BCbCFDb18564f3674DCF04EdEEe42bb747d'
```

Comment:
These are script specific (also the keys) - should not be included in general .env. People who use the script can read the env variables from the script itself anyway